docs/_documentations/troubleshooting.md

```diff
@@ -309,6 +309,18 @@ If you modify files in MicroProfile projects, sometimes the project gets double
 ```xml
 <applicationMonitor pollingRate="1000ms" />
 ```
+
+<!--
+Action/Topic: Checking the application and build statuses
+Issue type: info
+Issue link: https://github.com/eclipse/codewind/issues/1251
+Info added in 0.10.0.
+-->
+## Setting Codewind server logging levels
+To assist with problem determination, raise the default Codewind server log level to **Debug** or **Trace**. Use the `cwctl loglevels` command or follow the instructions for an IDE:
+- In Eclipse, set your preference to enable support features. Right-click the connection or the connection menu in the Codewind Explorer view. Now, the menus include setting the Codewind server log level items.
```

Comment:
How about:
In Eclipse, enable support features in the Codewind preferences, then right-click the connection in the Codewind Explorer view and select **Codewind server log level**.
I think we need to be clear about which preferences as Eclipse has a lot of preferences.